__tests__/app.test.js

```diff
@@ -125,37 +125,35 @@ describe("Articles", () => {
         });
     });
     test("adds sort by functionality default sorts columns by date", () => {
```

Comment:
In the description you've said this should default to be sorted by date but this test doesn't actually test whether the responses are sorted by date.